libyul/backends/evm/EVMDialect.cpp

```diff
@@ -342,6 +360,38 @@ SideEffects EVMDialect::sideEffectsOfInstruction(evmasm::Instruction _instructio
 	};
 }
 
+BuiltinFunctionForEVM const* EVMDialect::verbatimFunction(size_t _arguments, size_t _returnVariables) const
+{
+	pair<size_t, size_t> key{_arguments, _returnVariables};
+	shared_ptr<BuiltinFunctionForEVM const>& function = m_verbatimFunctions[key];
+	if (!function)
+	{
+		BuiltinFunctionForEVM builtinFunction = createFunction(
+			"verbatim_" + to_string(_arguments) + "_" + to_string(_returnVariables),
+			1 + _arguments,
+			_returnVariables,
+			SideEffects::worst(),
+			vector<optional<LiteralKind>>{LiteralKind::String} + vector<optional<LiteralKind>>(_arguments),
+			[=](
+				FunctionCall const& _call,
+				AbstractAssembly& _assembly,
+				BuiltinContext&,
+				std::function<void(Expression const&)> _visitExpression
+			) {
+				yulAssert(_call.arguments.size() >= 1, "");
+				for (Expression const& arg: _call.arguments | ranges::view::drop_exactly(1) | ranges::view::reverse)
+					_visitExpression(arg);
+				Expression const& bytecode = _call.arguments.front();
+				int diff = static_cast<int>(_returnVariables) - static_cast<int>(_arguments);
+				_assembly.appendVerbatim(asBytes(std::get<Literal>(bytecode).value.str()), diff);
+			}
+		).second;
+		builtinFunction.isMSize = true;
```

Comment:
The alternative would be to not use the convenience function `createFunction`

Reply:
I meant, setting `isMSize` to be true to avoid optimizer issues. Can you also add a comment before `MSizeFinder` that it would return true for `verbatim`?

Reply:
I wasn't sure either, but figured maybe some other builtin uses the same method?